README.md

```diff
@@ -2,7 +2,7 @@
 
 [![Build Status](https://cloud.drone.io/api/badges/tinkerbell/tink/status.svg)](https://cloud.drone.io/tinkerbell/tink)
 
-Tinkerbell is a bare metal provisioning and workflow engine comprised of three major components: a DHCP server ([boots](https://github.com/packethost/boots)), a workflow engine (tinkerbell, this repository), and a metadata service ([hegel](https://github.com/packethost/hegel)). The workflow engine is comprised of a server and a CLI, which communicate over gRPC. The CLI is used to create a workflow along with its building blocks, templates and targeted hardware.
+Tinkerbell is a bare metal provisioning and workflow engine. It was open-sourced by Packet, an Equinix company, and is comprised of four major components: a DHCP server ([boots](https://github.com/tinkerbell/boots)), a workflow engine (tink, this repository), a metadata service ([hegel](https://github.com/tinkerbell/hegel)) and an in-memory installation environment([osie](https://github.com/tinkerbell/osie)). The workflow engine is comprised of a server and a CLI, which communicate over gRPC. The CLI is used to create a workflow along with its building blocks, templates and targeted hardware.
```

Comment:
can you split this to 2 commits please? One to split each sentence onto its own line and the second with your Equinix and other changes?

Reply:
sure